### PR TITLE
Allow user to set certificate storage location

### PR DESCRIPTION
**Pull request by k3vmcd**
Also updates the Readme.md to include a more stable example of custom haproxy.cfg SSL redirect.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Let's Encrypt verification
 
-Let's Encrypt has two methods of verifying ownership of domains. The first is through the addition of a custom DNS record (say acme-12321313.subdomain.domain.com). This is what https://github.com/janeczku/rancher-letsencrypt does. That service creates Let's Encrypt challenges via DNS resolution. The other way of proving ownership of domains is through a webserver webroot over HTTP. 
+Let's Encrypt has two methods of verifying ownership of domains. The first is through the addition of a custom DNS record (say acme-12321313.subdomain.domain.com). This is what https://github.com/janeczku/rancher-letsencrypt does. That service creates Let's Encrypt challenges via DNS resolution. The other way of proving ownership of domains is through a webserver webroot over HTTP.
```
Comment (k3vmcd) [inline README.md:5]: This block shouldn't be marked changed, no content was updated. I think GitHub is picking up a formatting preference change.

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The service launches two containers:
 - `letsencrypt-nginx`
 - `letsencrypt-python`
 
-The `letsencrypt-nginx` container is stock nginx, but shares the webroot with the `letsencrypt-python` service container. This way the `letsencrypt-python` container can add ACME challenges to the `<host>/.well-known/acme-challenge/` directory on the webserver for verification. The python container is a sidekick of the nginx container. The containers are launched as a Rancher Service Account, so special environment variables containing the Rancher server API url, and access keys are passed into the container at runtime. 
+The `letsencrypt-nginx` container is stock nginx, but shares the webroot with the `letsencrypt-python` service container. This way the `letsencrypt-python` container can add ACME challenges to the `<host>/.well-known/acme-challenge/` directory on the webserver for verification. The python container is a sidekick of the nginx container. The containers are launched as a Rancher Service Account, so special environment variables containing the Rancher server API url, and access keys are passed into the container at runtime.
```
Comment (k3vmcd) [inline README.md:17]: This block shouldn't be marked changed, no content was updated. I think GitHub is picking up a formatting preference change.

File: README.md [hunk 3]
```diff
@@ -48,7 +47,7 @@ frontend http-frontend
 
 ## How to use
 
-Create a front end load balancer (or use the one in `traffic-manager` directory). If you are making one, you need to make sure it is a L7 HTTP load balancer on your chosen privileged port. This way the load balancer can redirect /.well-known/\* traffic to the `letsencrypt-nginx` container for verification. You can then route all other traffic to your normal HTTP services. This way only during verification does traffic get directed to the `letsencrypt-nginx` container. 
+Create a front end load balancer (or use the one in `traffic-manager` directory). If you are making one, you need to make sure it is a L7 HTTP load balancer on your chosen privileged port. This way the load balancer can redirect /.well-known/\* traffic to the `letsencrypt-nginx` container for verification. You can then route all other traffic to your normal HTTP services. This way only during verification does traffic get directed to the `letsencrypt-nginx` container.
```
Comment (k3vmcd) [inline README.md:50]: This block shouldn't be marked changed, no content was updated. I think GitHub is picking up a formatting preference change.

**Review by ericmann — APPROVED** [review]

**Comment by fernviridian** [comment]
Thank you @k3vmcd for this contribution!

**Pull request merged**